packages/manager/apps/web/client/app/private-database/database/ram/update/private-database-database-ram-update.controller.js

```diff
@@ -167,5 +167,9 @@ angular.module('App').controller(
           this.loading.validation = false;
         });
     }
+
+    static isProrataDuration({ duration }) {
+      return /^upto/.test(duration);
```

Comment:
This could be handled by a generic model (perhaps extended depending on the service) 
something like: 
```suggestion
     class OrderableService {
       hasProrataDuration() {
         return /^upto/.test(this.duration)
        }
     }
```

Reply:
@marie-j I agree this could be a nice enhancement.

Since this ticket has already been validated I will merge it to the `develop` branch.

Feel free to open an Issue to address and track the reported suggestion.

Thanks